docs/02-app/02-api-reference/04-functions/revalidatePath.mdx

```diff
@@ -8,23 +8,50 @@ description: API Reference for the revalidatePath function.
 > **Good to know**:
 >
 > - `revalidatePath` is available in both [Node.js and Edge runtimes](/docs/app/building-your-application/rendering/edge-and-nodejs-runtimes).
-> - `revalidatePath` will revalidate _all segments_ under a dynamic route segment. For example, if you have a dynamic segment `/product/[id]` and you call `revalidatePath('/product/[id]')`, then all segments under `/product/[id]` will be revalidated as requested.
-> - `revalidatePath` only invalidates the cache when the path is next visited. This means calling `revalidatePath` with a dynamic route segment will not immediately trigger many revalidations at once. The invalidation only happens when the path is next visited.
+> - `revalidatePath` only invalidates the cache when the included path is next visited. This means calling `revalidatePath` with a dynamic route segment will not immediately trigger many revalidations at once. The invalidation only happens when the path is next visited.
 
 ## Parameters
 
 ```tsx
-revalidatePath(path: string): void;
+revalidatePath(path: string, type?: 'page' | 'layout'): void;
 ```
 
-- `path`: A string representing the filesystem path associated with the data you want to revalidate. This is **not** the literal route segment (e.g. `/product/123`) but instead the path on the filesystem (e.g. `/product/[id]`).
+- `path`: A string representing the filesystem path associated with the data you want to revalidate. This is the literal route segment (for example, `/product/123`) not the path on the filesystem `/product/[slug]/page`. Must be less than 1024 characters.
```

Comment:
Here we have

> `path`: [...] This is [...] not the path on the filesystem `/product/[slug]/page`.

but right below we also have "Revalidating A Page Path" and "Revalidating A Layout Path" sections right below that show code examples of `revalidatePath('/blog/[slug]', ...)`. I think this is a bit confusing. Does `/blog/[slug]` still work? Does the "not the path on the filesystem" sentence mean `/blog/[slug]` does work but `/blog/[slug]/page` doesn't (which is the current behaviour)?

Reply:
The current behavior is a bug, `/blog/[slug]` is only meant to work when the second argument is provided specifying `page` or `layout`. 

By default when only one argument is provided it should be the URL path meant to be revalidated which is what the examples below are trying to show-case and explain.

Reply:
I see, thanks!